src/components/ui/AppQrCode.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <qrcode-vue
+    :value="value"
+    :size="size"
+    level="H"
```

Comment:
An error correction of `L` probably suffices here, considering this will be displayed on a monitor and not printed out where it could be damaged. IMO it makes the QR code look a bit nicer, but I'm not sure it matters